src/marqo/tensor_search/configs.py

```diff
@@ -39,6 +39,8 @@ def default_env_vars() -> dict:
         EnvVars.MARQO_MAX_CONCURRENT_SEARCH: 8,
         EnvVars.MARQO_THREAD_EXPIRY_TIME: 1800,     # 30 minutes
         EnvVars.MARQO_ENABLE_THROTTLING: "TRUE",
-        EnvVars.MARQO_LOG_LEVEL: "info"             # This env variable is set to "info" by default in run_marqo.sh, which overrides this value
+        EnvVars.MARQO_LOG_LEVEL: "info",           # This env variable is set to "info" by default in run_marqo.sh, which overrides this value
+        EnvVars.MARQO_MAX_CPU_MODEL_MEMORY: 4,
+        EnvVars.MARQO_MAX_CUDA_MODEL_MEMORY: 4,
```

Comment:
Make sure the cloud team knows they have to manually set these values, so they don't get any surprises